WordPress/Classes/ViewRelated/Blog/Blogging Reminders/BloggingRemindersFlowSettingsViewController.swift

```diff
@@ -227,31 +252,67 @@ class BloggingRemindersFlowSettingsViewController: UIViewController {
     private func populateCalendarDays() {
         daysOuterStackView.addArrangedSubviews([daysTopInnerStackView, daysBottomInnerStackView])
 
-        let topRow = days[0..<4]    // First 4 days
-        let bottomRow = days[4..<days.count]    // Last 3 days
-        daysTopInnerStackView.addArrangedSubviews(topRow.map({ CalendarDayToggleButton(weekday: $0) }))
-        daysBottomInnerStackView.addArrangedSubviews(bottomRow.map({ CalendarDayToggleButton(weekday: $0) }))
-    }
-
-    /// Localized short weekday names, starting at the correct weekday for the current calendar
-    private let days: [String] = {
-        var calendar = Calendar.current
-        calendar.locale = Locale.autoupdatingCurrent
-        let firstWeekday = calendar.firstWeekday
-        var symbols = calendar.shortWeekdaySymbols
+        let createButton: (Int) -> CalendarDayToggleButton = { [unowned self] dayIndex in
+            let isSelected: Bool
+            let localizedDayIndex = (dayIndex + calendar.firstWeekday - 1) % calendar.shortWeekdaySymbols.count
```

Comment:
I wonder if this would be good moved to a small calendar extension?

Reply:
I'm not sure there's value in doing this as the resulting index aims specifically to match a `rawValue` of `BloggingRemindersScheduler.Weekday`.